### ISSUE TITLE
Since last update nothing works right...

### ISSUE THREAD

**Issue by taimutorabera** [tlovett1/custom-contact-forms#81]
Hi,
if i try to view submissions, there is only the actual date and time, if i click on to the eye-symbol, "Form Submission - Invalid date".
Over shortcut insertion i can only find an form with no title and if i try to insert, id is null.
Making a new form doesn't help too, after saving and reopening it is empty!
In the database i can finde the submission data.
Dont know whats wrong, but im sure it's the last update.

**Comment by tlovett1** [comment]
Did you upgrade your database?

**Comment by taimutorabera** [comment]
No...how I do it?

**Comment by tlovett1** [comment]
Please email tlovett88 [at] gmail

**Comment by taimutorabera** [comment]
Hi,
Emailing you cause the problems with my ccf backend after update.

Greets Taimutorabera
Am 07.03.2015 18:35 schrieb "Taylor Lovett" <notifications@github.com>:

**Comment by ronjo84** [comment]
Any update on this? I just updated to 6.5.1 and I can't seem to get any of my contact forms to load. I am not sure how to upgrade the database. Any help would be greatly appreciated.

**Comment by steveperrycreative** [comment]
As above. I've updated to the latest version and my form data has disappeared. It only shows a date. How do you run a database update for this?

**Comment by ronjo84** [comment]
Steve,

As an update, it looks like there should be an update database button after you update the plugin that will appear, if needed. It didn't appear for me, so I assume it isn't needed. 

My other issue was that the form wasn't loading after the update. I realized that is because the IDs changed. I had to go and re-insert the forms into my posts. 

For example, my form use to be: [customcontact form=5] and now it is [ccf_form id="2737"]. Hope that helps you.
